app/controllers/v0/claims_controller.rb

```diff
@@ -9,6 +9,16 @@ def index
              each_serializer: ClaimSerializer
     end
 
+    def documents
+      params.require :file
```

Comment:
`params.require` will throw an error if the key isn't present in the params hash, I think you could go either way in this case (given it should error out somewhere else if there is no file). I haven't seen `params.require` used much outside of strong parameters (whitelisting params before mass assignment)